bigbluebutton-html5/imports/ui/components/user-list/service.js

```diff
@@ -211,8 +214,15 @@ const getOpenChats = (chatID) => {
 const getAvailableActions = (currentUser, user, router, isBreakoutRoom) => {
   const hasAuthority = currentUser.isModerator || user.isCurrent;
   const allowedToChatPrivately = !user.isCurrent;
-  const allowedToMuteAudio = hasAuthority && user.isVoiceUser && !user.isMuted;
-  const allowedToUnmuteAudio = hasAuthority && user.isVoiceUser && user.isMuted;
+  const allowedToMuteAudio = hasAuthority
+                            && user.isVoiceUser
+                            && !user.isMuted
+                            && !user.isListenOnly;
```

Comment:
It's a good catch, I didn't know that we had this bug.
Turned out that currently a "Mute User" option is shown to moderator even when the user is in Listen Only.